src/Microsoft.AspNet.Mvc.Core/RedirectToRouteResult.cs

```diff
@@ -77,7 +78,14 @@ public override void ExecuteResult(ActionContext context)
 
         private IUrlHelper GetUrlHelper(ActionContext context)
```

Comment:
I was curious whether `GetRequiredService()` could be expensive enough and called frequently enough that this code should update the `UrlHelper` property. My expectation was that would be useless because the result is executed just once.